### DISCUSSION TITLE
Getting Wrong Device Specifiction

### DISCUSSION THREAD

**Discussion by ShashikantMauryaOnbe** [faisalman/ua-parser-js#647] [General]
Hi All,

I am using ua-parser to get device details, its working for most of the Samsung devices but unable to get device details when I am parsing Samsung S21.  

When I am trying to **google what is my User Agent** it gives like and seems to correct : **Mozilla/5.0 (Linux; Android 13; SM-G991U Build/TP1A.220624.014; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/111.0.5563.116 Mobile Safari/537.36 GoogleApp/[14.12.16.26](http://14.12.16.26/).arm64**

but **UA parser user agent** returned like : **Mozila/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Mobile Safari/537.36**

![image](https://user-images.githubusercontent.com/119935277/232750172-90381395-a6fa-4302-836b-f08f1b40fa0a.png)

please help me out.

**Comment by faisalman** [comment]
Try visit https://uaparser.js.org/ is the result still the same?

**Reply by faisalman** [reply]
New site: https://uaparser.dev
